.eslintrc.js

```diff
@@ -39,7 +39,8 @@ module.exports = {
     'no-prototype-builtins': 'off',
     'quote-props': 'off',
     'arrow-body-style': 'off',
-    'import/no-extraneous-dependencies': 'off'
+    'import/no-extraneous-dependencies': 'off',
+    'dot-notation': 'off'
```

Comment:
Just out of curiosity, why do you want to disable this rule?

Reply:
Instead of changing `this.services['events']` to `this.services.events`

Reply:
I personally like this one more, but I can go with both. But anyway I don't see a point in this rule, it restricts you too much IMO